### DISCUSSION TITLE
How to get the original text that was detected as an anonymization candidate in the Analyse phase?

### DISCUSSION THREAD

**Discussion by storkeu** [microsoft/presidio#925] [Q&A] [answered]
Hello, is it possible to see the source code for https://presidio-demo.azurewebsites.net/ ?
I'm interested in understanding how the original text that was detected as an anonymization candidate entity is printed in the findings table (column Text). I need to show my users the original text but I don't know where to get this info.
Thank You

**Comment by omri374** [comment, marked as answer]
Hi @storkeu, we have a similar demo website in our samples gallery: https://microsoft.github.io/presidio/samples/python/streamlit/

Programmatically, one can access the detected entity value this way:
```python
from presidio_analyzer import AnalyzerEngine

text="My phone number is 212-555-5555"
analyzer = AnalyzerEngine()

# Call analyzer to get results
results = analyzer.analyze(text=text, language='en')

for result in results:
    print(f"Text: {text[result.start: result.end]} identified as entity {result.entity_type}")
```
Returning:
```
Text: 212-555-5555 identified as entity PHONE_NUMBER
```

**Reply by storkeu** [reply]
So simple!. Tks!

**Comment by rabbyn** [comment]
Hello @omri374 , was the ready made demo website https://presidio-demo.azurewebsites.net/  decommissioned ? that's pity that was good to showcase the features to business users directly :(

**Reply by omri374** [reply]
Hi #rabbyn, yes it was, but we have a different one hosted on HuggingFace hub that you can use (and clone): https://huggingface.co/spaces/presidio/presidio_demo

**Reply by rabbyn** [reply]
thanks @omri374 ! I just shared the Hugging Face link to my colleagues (PS : maybe it will be good to have this link shown somewhere in the docs or readme.md)

**Reply by omri374** [reply]
Done :) https://github.com/microsoft/presidio/pull/1068
